### DISCUSSION TITLE
Where will old deta services go?

### DISCUSSION THREAD

**Discussion by ndaidong** [deta/cloud-docs#683] [General] [answered]
Hi team,

Deta cloud system includes 2 parts: `deta.sh` which I call "old" services and `deta.space` which is still in its alpha stage.

Old services with Base, Drive and Micro are so cool for backend development. Personally I prefer using them flexibly without building "installable apps" as I have to with `deta.space`. 

What are your plans for the future of `deta.sh`, once `deta.stace` gets official release?

**Comment by sudo-subg** [comment, marked as answer]
Currently the team is working on Deta Space(v2) and Cloud(v1) merger. Deta Space will have all the features of Deta Cloud. As of now some of the features are missing in Deta Space but those features will be added soon after merger. Till that point devs will have a time to migrate their projects from Deta Cloud to Space.
They have plans for public APIs and apps so you will be able to use them as before with more flexibility like custom commands, more runtimes, frameworks supports out of the box and some cool new features which the team has plans for.
No ETA for most of these things expect the merger which is going to be pretty soon afaik.
I think this answers all your queries but lemme know if I am missing something. Lastly, they are a small team, so please be patient 🙂.

**Comment by ndaidong** [comment]
@mikBighne98 thank you for your answer. Wish the team will grow more and more :)
